test/api_integration/apis/core/index.js

```diff
@@ -32,5 +32,11 @@ export default function ({ getService }) {
         .get('/requestcontext/savedobjectsclient')
         .expect(200, 'SavedObjects client: {"page":1,"per_page":20,"total":0,"saved_objects":[]}')
     ));
+
+    it('provides access to application rendering client', async () => {
+      await supertest.get('/requestcontext/render/core').expect(200, /app:core/);
+      await supertest.get('/requestcontext/render/testbed').expect(200, /app:testbed/);
+      await supertest.get('/requestcontext/render/invalid').expect(404);
```

Comment:
does this test pass?

Reply:
Not yet, this is left in because we will be adding back the ID to rendering for legacy.